executor/infoschema_reader_test.go

```diff
@@ -309,3 +326,207 @@ func (s *testInfoschemaTableSuite) TestPartitionsTable(c *C) {
 
 	tk.MustExec("DROP TABLE `test_partitions`;")
 }
+
+var _ = SerialSuites(&testInfoschemaClusterTableSuite{testInfoschemaTableSuite: &testInfoschemaTableSuite{}})
```

Comment:
Do we realy need to nest the `testInfoschemaTableSuite` in the `testInfoschemaClusterTableSuite `?

Reply:
unneccessaryly, just for reuse code like done in the tables_test.go before, it is ok to decouple.